src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/DefaultObjectValidator.cs

```diff
@@ -39,10 +39,12 @@ internal class DefaultObjectValidator : ObjectModelValidator
                 validatorProvider,
                 validatorCache,
                 metadataProvider,
-                validationState);
-
-            visitor.MaxValidationDepth = _mvcOptions.MaxValidationDepth;
-            visitor.AllowShortCircuitingValidationWhenNoValidatorsArePresent = _mvcOptions.AllowShortCircuitingValidationWhenNoValidatorsArePresent;
+                validationState)
+            {
+                MaxValidationDepth = _mvcOptions.MaxValidationDepth,
+                AllowShortCircuitingValidationWhenNoValidatorsArePresent =
```

Comment:
Might have missed when this was discussed in the issue, but would we keep this? I thought we would just default to doing this in 3.0. The flag was there because we just didn't feel confident enough about enabling the behavior super late in 2.2

Reply:
The change here is just cleanup. `AllowShortCircuitingValidationWhenNoValidatorsArePresent` goes away in part 3 of 3.